pandas-stubs/core/strings.pyi

```diff
@@ -66,8 +69,13 @@ class StringMethods(NoNewAttributesMixin, Generic[T, _TS, _TM]):
     ) -> _TS: ...
     @overload
     def split(
-        self, pat: str = ..., *, n: int = ..., expand: bool = ..., regex: bool = ...
-    ) -> T: ...
+        self,
+        pat: str = ...,
+        *,
+        n: int = ...,
+        expand: Literal[False] = ...,
+        regex: bool = ...,
+    ) -> _TS2: ...
     @overload
     def rsplit(self, pat: str = ..., *, n: int = ..., expand: Literal[True]) -> _TS: ...
```

Comment:
Just noticed this.  You should do `rsplit()` as well for completeness (including tests)